bin/kano-uixinit-systemd

```diff
@@ -21,61 +22,10 @@
 
 logger --id --tag "info" "uixinit systemd starts"
 
-# FIXME: is this still needed?
-mkdir -p "$HOME/.kano-settings"
-
-# Track some information about the hardware used
-kano-tracker-ctl generate hw-info
-
-# Disable XServer screen saver time and screen blanking (The display would become black)
-if [ -n "`which xset`" ]; then
-    xset s off
-    xset -dpms
-    xset s noblank
-fi
-
-# Calibrates the mouse pointer speed
-/usr/bin/startmouse
-
-# Set the initial default volume
```

Comment:
If you just remove it though, does the volume persist across boot? I think we have this code here because lxpanel sets it to a fixed value and we want to restore the one the user set.